lib/vehicle.rb

```diff
@@ -7,12 +7,17 @@ class Vehicle
               :model,
               :engine
 
+  attr_accessor :registration_date,
+                :plate_type
```

Comment:
Do we need plate_type here to be an attr_accessor? You have the method of `set_plate_type_dawggy` (were you tired by this point? :) ) which handles setting the plate type for you.